### DISCUSSION TITLE
Why db type and charset is ommited in cli install?

### DISCUSSION THREAD

**Discussion by quentincaffeino** [directus/directus#2856] [Archive]
https://github.com/directus/directus/blob/5bdfc79eef6a82f8f075af27880f2a5fb85a7bee/src/core/Directus/Console/Modules/InstallModule.php#L178-L198

**Comment by quentincaffeino** [comment]
I tried using directus with postgres but right now it seems that this requires a lot changes to the core code. I've seen an issue. This discussion can be removed.
Thanks.

**Reply by benhaynes** [reply]
PostgreSQL (as well as MySQL, MS-SQL, OracleDB, SQLite, and more) will all be officially supported in the upcoming Directus 9 release, which will be production ready within the next month or so.

**Reply by quentincaffeino** [reply]
I already tried directus-next (node), nice work! Even with a couple little bugs more than enough for using for myself)
